src/functions-and-arrays.js

```diff
@@ -52,15 +107,37 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+
+  let newArray = [];
+
+  for (let i = 0; i < wordsUnique.length; i++) {
+
+    if (newArray.includes(wordsUnique[i])) {
```

Comment:
You could also check if the array doesn't include the word by using `!newArray.includes(wordsUnique[i])`, then you don't need an else